src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -200,6 +202,16 @@ private static List<String> getTablesFromEntitySet(Set<? extends Entity> entitie
     return tableNames;
   }
 
+  private static List<String> getDatabasesFromEntitySet(Set<? extends Entity> entities) {
```

Comment:
This dumps all databases touched by a query. Just to confirm – do you plan to add the database, which was set as current for the user at the moment of running the query?

Reply:
will add database in separate pr